src/openrct2/object/ObjectManager.cpp

```diff
@@ -503,11 +503,6 @@ class ObjectManager final : public IObjectManager
                 sgObject->UpdateEntryIndexes();
             }
         }
-
-        // HACK Scenery window will lose its tabs after changing the scenery group indexing
-        //      for now just close it, but it will be better to later tell it to invalidate the tabs
-        auto* windowMgr = Ui::GetWindowManager();
-        windowMgr->CloseByClass(WindowClass::Scenery);
```

Comment:
I think this is no longer necessary. We could keep it around as well, but then I'd need to find a way to prevent it from being called while UiContext hasn't been initialised yet.